### ISSUE TITLE
Large arrays in method arguments 

### ISSUE THREAD

**Issue by mvriel** [phpDocumentor/phpDocumentor#2228]
@jrfnl reported this during #phpbnl20:

In the `default` template: when a method signature contains a large array as the default value for an argument, the argument is not exactly readable anymore.

![image](https://user-images.githubusercontent.com/193704/73127503-93da0b80-3fc1-11ea-8647-3b91ef1ab66e.png)

Since we lose most of the formatting information during parsing it is not possible to reconstruct the formatting of the original project and a different representation may be needed here. Either by neatly formatting the argument; or by cutting it and showing the full value, formatted, in the description below

**Comment by jrfnl** [comment]
@mvriel Would it help if I did a full write-up with example code for this like I've done for other issues recently ?

**Comment by mvriel** [comment]
@jrfnl I do not think so at the moment, the issue is clear to me and a reproduction scenario is easy. I am still figuring out what a good format would be since this issue also affects the default values for parameters in signatures. I am considering moving the information from the signature to a special field in the description so that it is easier and more readable to format

**Comment by jrfnl** [comment]
@mvriel Ok. Feel free to let me know if you'd like me to review something or test something related to this.
